README.md

```diff
@@ -1,7 +1,7 @@
 clj-multihash
 =============
 
-[![Dependency Status](https://www.versioneye.com/clojure/mvxcvi:multihash/badge.svg)](https://www.versioneye.com/clojure/mvxcvi:multihash)
+[![Dependency Status](https://www.versioneye.com/clojure/mvxcvi:multihash/2.0.1/badge.svg)](https://www.versioneye.com/clojure/mvxcvi:multihash)
```

Comment:
Yeah. This bot had a few issues (little, comparatively). Will remove this from this PR.